src/sentry/static/sentry/app/views/projectsDashboard/resources.tsx

```diff
@@ -11,9 +12,18 @@ import releasesImg from '../../../images/spot/releases.svg';
 import breadcrumbsImg from '../../../images/spot/breadcrumbs-generic.svg';
 import docsImg from '../../../images/spot/code-arguments-tags-mirrored.svg';
 
-export default class Resources extends React.Component {
+type Props = {
+  organization: Organization;
+};
+
+class Resources extends React.Component<Props> {
   componentDidMount() {
-    analytics('orgdash.resources_shown');
+    const {organization} = this.props;
+    trackAnalyticsEvent({
+      eventKey: 'orgdash.resources_shown',
```

Comment:
Is the eventKey the same as `analytics('orgdash.resources_shown');`, does it matter?

Reply:
They look the same to me :+1: